script/test.py

```diff
@@ -0,0 +1,35 @@
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+from collections import namedtuple
+from collections import Counter
+from main import find_links
+
+
+LinkInfo  = namedtuple('LinkInfo', 'name href')
+
+
+def test_find_links(driver):
+  lists_of_tuples = find_links(driver)
+  for LinkInfo.href in lists_of_tuples:
+    if "href" == None:
+      assert Error in lists_of_tuples      
+    else:
+      assert "All tests done."  
+  return lists_of_tuples
+
+
+def test_count_links(links):
```

Comment:
This function should be self sufficient. It means that it should:

  * Get the page: driver.get("https://annalangner.github.io/Projekt-1/")
  * find_links
  * Verify number of links

So the parameter here should be a driver not list of links